### ISSUE TITLE
Consider renaming this repo

### ISSUE THREAD

**Issue by cmungall** [HOT-Ecosystem/tccm-model#7]
Having both tccm and tccm-model is confusing

maybe call the latter tccm-lite?

or maybe the two have diverged enough that they should no longer have the same base name

**Comment by joeflack4** [comment]
I agree it is confusing.

I think that if we keep `tccm` named as `tccm`, `tccm-model` --> `tccm-lite` is not a bad suggestion. I am reluctant to make the change without @hsolbrig approval.

I'm often not a fan of `<org>/<repo>` where `<org>` and `<repo>` have the same name. I'm generally fine with this if the repo of the same name has no content in it, and is just used for organizing issues, etc. Or, if it simply has documentation (although in that case adding `-documentation` to the name might be better than leaving the repo name the same as the org.

@cmungall This is actually an issue for me with LinkML too. If you go to [`linkml/linkml`](https://github.com/linkml/linkml), there is actually a problem to `tccm/tccm`. Actually, I'm still not sure what this repository is for. I understand that the issues there are broadly pertaining to LinkML, but I don't understand what the contents of the repo are for; what purpose the repository serves, and the README doesn't help there either (although it does link to docs, which is good).

**Comment by cmungall** [comment]
Hmm, it’s not good if you don’t know what the linkml repo is for, can you
make an issue (in linkml!) for that?

Should we flesh this out
https://linkml.io/linkml/developers/organization.html

**Comment by joeflack4** [comment]
@cmungall Yes, my apologies for my ignorance, but maybe it's a good thing in this case. I'll make an issue.

I'm not sure fleshing out that page is necessary. I think it's better to consider what LinkML looks like outside of our small developers sphere. I'm thinking in the context of promoting adoption. I'm not sure what kinds of search queries might lead people to find LinkML, but I think that since we do want to encourage its eventual more widespread use, it would be good for the purpose of each repo to be clear, even in the case where someone first arrives at the repo with no prior context.

I created the issue here: https://github.com/linkml/linkml/issues/762
